deepfence_ui/app/scripts/charts/node-action-modal.js

```diff
@@ -2,28 +2,33 @@
 /* eslint-disable no-unused-vars */
 /* eslint-disable max-len */
 import React from 'react';
-import { useDispatch } from 'react-redux';
+import { useDispatch, connect } from 'react-redux';
+import {formValueSelector} from 'redux-form/immutable';
 import {
   scanRegistryImagesAction,
 } from '../actions/app-actions';
 import CVEScanForm from '../components/vulnerability-view/registry-scan/cve-scan-form';
 import { nodeListWithType } from '../components/multi-cloud-table/utils';
 
-export const NodeActionModal = ({selectedDocIndex = {}, resetSelection, isCVE}) => {
+const NodeActionModal = (props) => {
+  const {selectedDocIndex = {}, resetSelection, isCVE, priorityValue} = props
   const dispatch = useDispatch();
 
   const startBulkCVEScan = (params) => {
+    const priorityValueCheck = !!(priorityValue && priorityValue[0] === 'priority');
```

Comment:
this check can be better, check if priority value is an array and its length is more than 0